jsapp/js/components/transferProjects.component.tsx

```diff
@@ -0,0 +1,279 @@
+          <section className={styles.modalBody}>
+            <p>
+              {t('This action will transfer ownership of')}
+              &nbsp;
+              {props.asset.name}
+              &nbsp;
+              {t('to another user.')}
+            </p>
+            <p>
+              {t(
+                'When you transfer ownership of the project to another user, all of the submissions, data storage, and transcription and translation usage for the project will be transferred to the new project owner.'
+              )}
+            </p>
+            <p>
+              {t(
+                'The new project owner will receive an email request to accept the transfer. You will be notified when the transfer is accepted or declined.'
+              )}
+            </p>
+
+            <div className={styles.warning}>
+              <Icon
+                name='warning'
+                size='s'
+                color='red'
+                classNames={[styles.warningIcon]}
+              />
+
+              <div className={styles.warningCopy}>
+                {t(
+                  'You will be the owner of the project until the transfer is accepted.'
+                )}
+                <br />
+                <strong>
+                  {t(
+                    'Once the transfer is accepted, you will not be able to undo this action.'
+                  )}
+                </strong>
+                &nbsp;
+                <a
+                  href={
+                    envStore.data.support_url +
+                    HELP_ARTICLE_ANON_SUBMISSIONS_URL
+                  }
+                  target='_blank'
+                >
+                  {t('Learn more')}
+                </a>
+              </div>
+            </div>
+            {/* Unused element to prevent firefox autocomplete
+            suggestions on username field */}
+            <input type='text' style={{display: 'none'}} />
+            <div className={styles.input}>
+              <TextBox
+                label={t(
+                  'To complete the transfer, enter the username of the new project owner'
+                )}
+                type='text'
+                value={transfer.usernameInput}
+                placeholder={t('Enter username here')}
+                required
+                errors={transfer.usernameError}
+                onChange={updateUsername}
+              />
+            </div>
+          </section>
+
+          <KoboModalFooter alignment='end'>
+            <Button
+              label={t('Cancel')}
+              onClick={toggleModal}
+              color='blue'
+              type='frame'
+              size='m'
+            />
+            <Button
+              label={t('Transfer project')}
+              onClick={(evt: React.FormEvent<HTMLFormElement>) => {
+                evt.preventDefault();
+                submitInvite(transfer.usernameInput);
+              }}
+              isPending={transfer.submitPending}
```

Comment:
If I have 500 error when sending invite, there is no coming back - even closing modal doesn't remove pending status

Reply:
And I got 500 error while doing this:
1. I am logged in as "joe" (admin)
1. I use `/admin` to log in as different user
2. I try transfering project to "joe"

I get this:


```
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/django/contrib/staticfiles/handlers.py", line 76, in __call__
    return self.application(environ, start_response)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/wsgi.py", line 133, in __call__
    response = self.get_response(request)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 130, in get_response
    response = self._middleware_chain(request)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 49, in inner
    response = response_for_exception(request, exc)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception
    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 149, in handle_uncaught_exception
    return debug.technical_500_response(request, *exc_info)
  File "/opt/venv/lib/python3.10/site-packages/django_extensions/management/technical_response.py", line 40, in null_technical_500_response
    raise exc_value.with_traceback(tb)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/opt/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/venv/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/mixins.py", line 19, in create
    self.perform_create(serializer)
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/mixins.py", line 24, in perform_create
    serializer.save()
  File "/opt/venv/lib/python3.10/site-packages/rest_framework/serializers.py", line 212, in save
    self.instance = self.create(validated_data)
  File "/srv/src/kpi/kobo/apps/project_ownership/serializers/invite.py", line 89, in create
    self._send_invite_email(instance)
  File "/srv/src/kpi/kobo/apps/project_ownership/serializers/invite.py", line 305, in _send_invite_email
    Mailer.send(email_message)
  File "/srv/src/kpi/kpi/utils/mailer.py", line 71, in send
    send_mail(
  File "/opt/venv/lib/python3.10/site-packages/django/core/mail/__init__.py", line 61, in send_mail
    return mail.send()
  File "/opt/venv/lib/python3.10/site-packages/django/core/mail/message.py", line 284, in send
    return self.get_connection(fail_silently).send_messages([self])
  File "/opt/venv/lib/python3.10/site-packages/django/core/mail/backends/smtp.py", line 102, in send_messages
    new_conn_created = self.open()
  File "/opt/venv/lib/python3.10/site-packages/django/core/mail/backends/smtp.py", line 62, in open
    self.connection = self.connection_class(self.host, self.port, **connection_params)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
OSError: [Errno 99] Cannot assign requested address
```